build.gradle.kts

```diff
@@ -19,13 +19,13 @@ allprojects {
         google()
         mavenLocal {
             content {
-                includeVersionByRegex(Dependencies.Groups.jellyfin, ".*", Dependencies.Versions.jellyfinSdkLocal)
+                includeVersionByRegex(JellyfinSdk.GROUP, ".*", JellyfinSdk.LOCAL)
```

Comment:
Why not `libs.versions.jellyfinSdkLocal`?

Reply:
Good point, I'll fix that. Forgot that this worked too.

Reply:
Nvm, that doesn't work. It fails the build.

Reply:
Maybe it needs Gradle 7.2 :(

Reply:
I'd merge it as is then.